tensorflow/core/kernels/dml_reduce_ops.cc

```diff
@@ -171,6 +171,19 @@ class DmlReduceKernel : public DmlKernel {
         reduce_function == DML_REDUCE_FUNCTION_MIN ||
         reduce_function == DML_REDUCE_FUNCTION_MAX;
 
+    // Arg functions in TF are defined with an int64 output, but the indices
+    // cannot be negative we output uint32 values with strides instead of int32
+    // with padding
```

Comment:
"since"

    // Arg functions in TF are defined with an int64 output, but since the indices
    // cannot be negative, we output uint32 values with strides instead of int32